Make.inc

```diff
@@ -81,38 +81,23 @@ ifneq (,$(findstring MINGW,$(OS)))
 override OS=WINNT
 endif
 
-#keep these if statements separate
```

Comment:
that also should not have removed the comment. the "separate" part is referring to
```
ifneq (,$(findstring MINGW,$(OS)))
 override OS=WINNT
 endif
 
 #keep these if statements separate
```